nevergrad/benchmark/experiments.py

```diff
@@ -1162,11 +1162,11 @@ def arcoating(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
 
 
 @registry.register
-def image_similarity(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+def image_similarity(seed: tp.Optional[int] = None, with_pgan: bool = False) -> tp.Iterator[Experiment]:
     """Optimizing images: artificial criterion for now."""
     seedg = create_seed_generator(seed)
     optims = get_optimizers("structured_moo", seed=next(seedg))
-    func = imagesxp.Image()
+    func = imagesxp.Image(with_pgan=with_pgan)
     for budget in [100 * 5 ** k for k in range(3)]:
         for num_workers in [1]:
```

Comment:
Maybe the loop on num_workers is not necessary ?

Reply:
Removed